RNA-seq/05-nb_cell_line_DESeq2.Rmd

```diff
@@ -298,7 +302,7 @@ dev.off()
 We can specify the dimensions and resolution of the PNG that is saved by 
 supplying different arguments to the `png` function.
 
-```{r}
+```{r png-adjust}
```

Comment:
This section isn't exactly like it was before because exrcise doesn't yet allow for chunks to be both exrcised and partially typed in. https://github.com/AlexsLemonade/exrcise/issues/3